src/routes/user.router.js

```diff
@@ -0,0 +1,7 @@
+const routes = require("express").Router();
+const userController = require("../controllers/user.controller");
+const { checkAuth } = require("../utils/checkAuth")
+
+routes.get("/:userId", checkAuth, userController.getUserDetail);
```

Comment:
```suggestion
routes.get("", checkAuth, userController.getUserDetail);
```
userId를 따로 받을 필요가 있을까요? 어차피 본인 정보를 수정할테니 토큰에 있는 유저 정보를 활용할 수 있겠습니다.

Reply:
멘토님, userId는 본인 상점정보 뿐만 아니라 다른 상점(유저) 정보 불러오기에도 사용되므로 필요합니다